routes/teamMember.js

```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+
+const router = express.Router();
+
+const TeamMember = require('../controllers/team_member');
+
+// TeamMember routes:
+
+// create TeamMember
+
+router.post('/team-member/add', TeamMember.createTeamMember);
+
+// update TeamMemer
+
+router.put('/team-member/:id', TeamMember.updateTeamMember);
+
+// Fetch all TeamMembers
+
+router.get('/team-members', TeamMember.getAllTeamMembers);
+
+// delete TeamMember
+
+router.delete('/team-member/:id', TeamMember.deleteTeamMember);
+
+module.exports = router;
```

Comment:
👍 
We can add route for getting single team member. We may use it in future.

Reply:
Yes, that was something I was wondering about, I'll get that implemented!

Reply:
Resolved with a commit, refactored the controller method and added a route